extensions/virtuallistview/src/VirtualListView.tsx

```diff
@@ -1107,17 +1143,25 @@ export class VirtualListView<ItemInfo extends VirtualListViewItemInfo>
         }
 
         _.each(cellList, cell => {
-            let tabIndexValue: number|undefined;
+            let tabIndexValue = -1;
             let isFocused = false;
+            let isSelected = false;
             if (cell.item) {
                 if (cell.item && cell.item.isNavigable) {
```

Comment:
The first part of this conditional is redundant. (I know this was in the existing code, not in your changes.)